helloworldmtls/helloworld.go

```diff
@@ -76,6 +76,7 @@ func ParseClientOptionFlags(args []string) (client.Options, error) {
 			return client.Options{}, fmt.Errorf("server CA PEM file invalid")
 		}
 	}
+	// @@@SNIPEND
```

Comment:
I wonder if we really need the custom server CA cert stuff. I did it here because it makes sense for self-hosted Temporal servers, but for cloud you probably won't need it. I believe our certs are signed by legit root CAs.

Reply:
You don't even need `serverCAPool` except if you're hosting the server using a self-signed certificate (not to be confused with a self-signed client certificate). But I'll let @bergundy weigh in on whether we want to document how to do server CA override.

Reply:
I'd avoid this, I expect most of our users to not have to set this and work with proper CAs.